internal/controller/management/projects/projects.go

```diff
@@ -428,6 +432,17 @@ func (r *reconciler) syncProject(
 		return *status, fmt.Errorf("error ensuring default project roles: %w", err)
 	}
 
+	if err := r.ensureReceiversFn(ctx, project); err != nil {
```

Comment:
It feels out of place to be doing this here. This reconciler is for Projects. The webhook receiver config is part of ProjectConfig. Those should really be reconciled separately.